test/integration/standard_filter_test.rb

```diff
@@ -353,9 +353,21 @@ def test_floor
   end
 
   def test_append
-    assigns = {'a' => 'bc', 'b' => 'd' }
-    assert_template_result('bcd',"{{ a | append: 'd'}}",assigns)
-    assert_template_result('bcd',"{{ a | append: b}}",assigns)
+    assigns = {'a' => 'bc', 'b' => 'd', 'c' => [1,2,3,4], 'd' => 10 }
+
+    assert_template_result('bcd'      ,"{{ a | append: 'd'}}" ,assigns)
+    assert_template_result('bcd'      ,"{{ a | append:  b }}" ,assigns)
+    assert_template_result('123410'   ,"{{ c | append:  d }}" ,assigns)
+    assert_template_result('bc1234'   ,"{{ a | append:  c }}" ,assigns)
+    assert_template_result('12341234' ,"{{ c | append:  c }}" ,assigns)
+
+    assert_equal([1,2,3,4],  @filters.append([1,2], [3,4] ))
+    assert_equal([1,2,'a'],  @filters.append([1,2],  'a'  ))
+    assert_equal([1,2, 10],  @filters.append([1,2],  10   ))
+    assert_equal(['a', 1,2], @filters.append('a',  [1,2]  ))
```

Comment:
IMO, this should behave like plan-ruby would, so nobody gets unexpected behaviour:
```ruby
irb(main):004:0> '1' << '2'
=> "12"
irb(main):005:0> '1' << ['2']
TypeError: no implicit conversion of Array into String
	from (irb):5
	from /usr/bin/irb:12:in `<main>'
irb(main):006:0> ['1'] << ['2']
=> ["1", ["2"]]
irb(main):007:0> ['1'] << '2'
=> ["1", "2"]
```

Those 4 cases should be the tests in here.